packages/react-router-dom/modules/Link.js

```diff
@@ -4,71 +4,89 @@ import PropTypes from "prop-types";
 import invariant from "tiny-invariant";
 import { resolveToLocation, normalizeToLocation } from "./utils/locationUtils";
 
+// React 15 compat
```

Comment:
If they're on an older version of React, forwardRef isn't available so we just use an identity function so that our code doesn't throw.

Reply:
Sounds good. In the release notes we can say something like "`<Link ref>` and `<NavLink ref>` now work if you're on React >= 16.3"